src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -90,6 +90,39 @@
           />
         </DetailsItem>
 
+        <DetailsItem
+          v-if="popupProps?.tx?.gasLimit"
+          :value="popupProps.tx.gasLimit"
+          :label="$t('transaction.gasLimit')"
+          data-cy="gas"
+        />
+        <DetailsItem
+          v-if="gasPrice"
+          :label="$t('pages.transactionDetails.gasPrice')"
+          data-cy="gas-price"
+        >
+          <template #value>
+            <TokenAmount
+              :amount="+aettosToAe(gasPrice)"
+              :symbol="AE_SYMBOL"
+              :protocol="PROTOCOLS.aeternity"
+            />
+          </template>
+        </DetailsItem>
+        <DetailsItem
+          v-if="gasCost"
+          :label="$t('transaction.gasCost')"
+          data-cy="gas-price"
```

Comment:
The same `data-cy="gas-price"` is used above for `gasPrice`. Is this intentional?